### PR TITLE
sc-17812-maple-js-add-mpluseractions-abis-addresses

### PR DESCRIPTION
**Pull request by Jakkc**
|       Type        |              Ticket              |
| :---------------: | :------------------------------: |
| Chore | [Link](https://app.shortcut.com/maplefinance/story/17812/maple-js-add-mpluseractions-abis-addresses) |

## Problem
- Add `MplUserActions` ABI to maple-js

## Solution
- Added remaining contracts from SyrupUtilsPrivate, to save having to do this multiple times.
- Reconfigured exports from `SyrupRouter` to `SyrupUtilsPrivate` to reflect smart contract repo groupings more accurately.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,73 +1,143 @@
...
+        "DebtLockerV2",
+        "DebtLockerV2Factory"
+      ]
     },
     "debtLockerV3": {
-      "contracts": ["DebtLockerV3"]
+      "contracts": [
+        "DebtLockerV3"
+      ]
     },
```
Comment (josadcha) [inline config.json:13]: @Jakkc do you have any custom prettier configs that override the project ones?

Comment (callum-hyland) [reply config.json]: Fixed

File: src/addresses/mainnet-dev.ts [hunk 2]
```diff
@@ -101,14 +102,16 @@ export default {
...
-  stSyrup: '0x0000000000000000000000000000000000000000',
-  SyrupToken: '0x0000000000000000000000000000000000000000',
-  SyrupTokenImplementation: '0x0000000000000000000000000000000000000000',
-  SyrupTokenInitializer: '0x0000000000000000000000000000000000000000',
-  SyrupTokenMigrator: '0x0000000000000000000000000000000000000000',
+  stSyrup: '0xc7E8b36E0766D9B04c93De68A9D47dD11f260B45',
+  SyrupDrip: '0x509712F368255E92410893Ba2E488f40f7E986EA',
+  SyrupToken: '0x643C4E15d7d62Ad0aBeC4a9BD4b001aA3Ef52d66',
+  SyrupTokenImplementation: '0x6eD767EBCfF51533E5181f7bf818F2b9bD767aec',
+  SyrupTokenInitializer: '0x9c9499edD0cd2dCBc3C9Dd5070bAf54777AD8F2C',
```
Comment (callum-hyland) [inline src/addresses/mainnet-dev.ts:112]:
```suggestion
  SyrupTokenInitializer: '0xfE4a4fd3bd2E0Eb400355aeF5Aa1752bC54B30FC',
```

File: src/addresses/mainnet-prod.ts [hunk 3]
```diff
@@ -101,14 +102,16 @@ export default {
...
-  stSyrup: '0x0000000000000000000000000000000000000000',
-  SyrupToken: '0x0000000000000000000000000000000000000000',
-  SyrupTokenImplementation: '0x0000000000000000000000000000000000000000',
-  SyrupTokenInitializer: '0x0000000000000000000000000000000000000000',
-  SyrupTokenMigrator: '0x0000000000000000000000000000000000000000',
+  stSyrup: '0xc7E8b36E0766D9B04c93De68A9D47dD11f260B45',
+  SyrupDrip: '0x509712F368255E92410893Ba2E488f40f7E986EA',
+  SyrupToken: '0x643C4E15d7d62Ad0aBeC4a9BD4b001aA3Ef52d66',
+  SyrupTokenImplementation: '0x6eD767EBCfF51533E5181f7bf818F2b9bD767aec',
+  SyrupTokenInitializer: '0x9c9499edD0cd2dCBc3C9Dd5070bAf54777AD8F2C',
```
Comment (callum-hyland) [inline src/addresses/mainnet-prod.ts:112]:
```suggestion
  SyrupTokenInitializer: '0xfE4a4fd3bd2E0Eb400355aeF5Aa1752bC54B30FC',
```

File: src/addresses/mainnet-dev.ts [hunk 4]
```diff
@@ -101,14 +102,16 @@ export default {
   PriceOracleUSDT: '0x0000000000000000000000000000000000000000',
   PriceOracleWBTC: '0xF808ec05c1760DE4794813d08d2Bf1E16e7ECD0B',
   PriceOracleWETH: '0x40db4461402125fd848ccf89f39166d42db0aa47',
-  RecapitalizationModule: '0x7D75cF9Aab6cB9598bB6d9Bd81BaAA288cecA9Bf',
+  RecapitalizationModule: '0x5dfe0460f66fa06bFCbB3211e723556be6B3f69D',
```
Comment (callum-hyland) [inline src/addresses/mainnet-dev.ts:105]:
```suggestion
  RecapitalizationModule: '0x6b1A78C1943b03086F7Ee53360f9b0672bD60818',
```

File: src/addresses/mainnet-prod.ts [hunk 5]
```diff
@@ -101,14 +102,16 @@ export default {
   PriceOracleUSDT: '0x0000000000000000000000000000000000000000',
   PriceOracleWBTC: '0xF808ec05c1760DE4794813d08d2Bf1E16e7ECD0B',
   PriceOracleWETH: '0x40db4461402125fd848ccf89f39166d42db0aa47',
-  RecapitalizationModule: '0x7D75cF9Aab6cB9598bB6d9Bd81BaAA288cecA9Bf',
+  RecapitalizationModule: '0x5dfe0460f66fa06bFCbB3211e723556be6B3f69D',
```
Comment (callum-hyland) [inline src/addresses/mainnet-prod.ts:105]:
```suggestion
  RecapitalizationModule: '0x6b1A78C1943b03086F7Ee53360f9b0672bD60818',
```

**Review by callum-hyland — APPROVED** [review]

**Pull request merged**